cmd/connector/main.go

```diff
@@ -53,7 +57,14 @@ func main() {
 }
 
 func startConnector(ctx *cli.Context) error {
-	cfg, err := loadConfig(configPath)
+	configFilePath := ctx.GlobalString(configFile.Name)
+
+	absoluteConfigPath, err := filepath.Abs(configFilePath)
```

Comment:
i don't think we need absolute path here, it should work also with relative path

Reply:
changed.